lang/go_resolvers_test.go

```diff
@@ -0,0 +1,82 @@
+package lang_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/safedep/code/core"
+	"github.com/safedep/code/fs"
+	"github.com/safedep/code/lang"
+	"github.com/safedep/code/parser"
+	"github.com/stretchr/testify/assert"
+)
+
+var goImportExpectations = []ImportExpectations{
+	{
+		filePath: "fixtures/imports.go",
+		imports: []string{
+			"ImportNode{ModuleName: \"fmt\", ModuleItem: , ModuleAlias: \"fmt\", WildcardImport: false}",
+			"ImportNode{ModuleName: \"github.com/safedep/code/parser\", ModuleItem: , ModuleAlias: \"github.com/safedep/code/parser\", WildcardImport: false}",
+			"ImportNode{ModuleName: \"os\", ModuleItem: , ModuleAlias: osalias, WildcardImport: false}",
+			"ImportNode{ModuleName: \"github.com/safedep/code/core\", ModuleItem: , ModuleAlias: codeccorealias, WildcardImport: false}",
+			"ImportNode{ModuleName: \"embed\", ModuleItem: , ModuleAlias: , WildcardImport: true}",
+			"ImportNode{ModuleName: \"math\", ModuleItem: , ModuleAlias: , WildcardImport: true}",
+			"ImportNode{ModuleName: \"crypto\", ModuleItem: , ModuleAlias: cryptoalias, WildcardImport: false}",
+			"ImportNode{ModuleName: \"github.com/gin-contrib/pprof\", ModuleItem: , ModuleAlias: , WildcardImport: true}",
+			"ImportNode{ModuleName: \"github.com/smacker/go-tree-sitter\", ModuleItem: , ModuleAlias: gotreesitteralias, WildcardImport: false}",
+			"ImportNode{ModuleName: \"net/http\", ModuleItem: , ModuleAlias: , WildcardImport: true}",
+			"ImportNode{ModuleName: \"strings\", ModuleItem: , ModuleAlias: \"strings\", WildcardImport: false}",
```

Comment:
Note - Tree sitter implementation provides modulenames with quotes by default - "net/http"
In import node, we aren't storing plain strings, but the entire sitter.Node received from sitter API, hence storing as it is.
The quotes will be handled when consuming resolve imports API, eg. depsusage plugin

Reply:
@OmkarPh I think we should strip the quotes at language implementation level. It will just make the job simpler for consumer plugins (e.g. depsusage). Else same strip quote logic will be duplicated across all plugins. What do you think?

Reply:
yes, my first thought was stripping it in language itself, but we're storing entire nodes which makes it difficult
am exploring the sitter library for any modifications / copy APIs

Reply:
@OmkarPh Then let us go ahead with storing the nodes only so that we don't have to make any major framework checks. Should we explore providing getter methods line `ImportNode#GetImportName()` etc. which takes care of language specific transformation if required?

Reply:
I was experimenting with options,
- Introduce Getters in ast.ImportNode 
This would introduce two things, every importNode must know which language it represents and language-specific transformation logic inside `core` which I think isn't preferred ?

- Introduce a new resolver ResolveImportContents for every language so that language specific transformation stays within that language_resolvers only

I prefer the latter one, WDYT

Reply:
Implemented in this PR - https://github.com/safedep/code/pull/22

Reply:
@OmkarPh On re-thinking about this, I think the double quote is fine in the names because thats what the language choose to define an import name i.e. as a double quoted string. Let us leave the transformation to client code that uses the names. Let us not overload the resolver interface for this.